src/Symfony/Component/Scheduler/Trigger/CallbackMessageProvider.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Trigger;
+
+use Symfony\Component\Scheduler\Generator\MessageContext;
+
+final class CallbackMessageProvider implements MessageProviderInterface, \Stringable
+{
+    private \Closure $callback;
+
+    public function __construct(callable $callback, private string $description = '')
```

Comment:
Please add `@param callable(MessageContext): iterable<object>` as phpdoc, to document the expected signature of the callable. This way, static analysis tools will be able to detect invalid usages.